hack/deploy-cluster-capz.sh

```diff
@@ -129,7 +129,7 @@ function create_workload_cluster() {
   fi
 
   echo "Waiting for the kubeconfig to become available"
-  timeout --foreground 1000 bash -c "while ! kubectl get secrets | grep ${CLUSTER_NAME}-kubeconfig; do sleep 1; done"
+  timeout --foreground 1000 bash -c "while ! kubectl get secrets -n "${CLUSTER_NAME}" | grep ${CLUSTER_NAME}-kubeconfig; do sleep 1; done"
```

Comment:
IRRC, the secret is in default namespace. Have you check it?

Reply:
This was the case earlier but it was updated with this change https://github.com/kubernetes-sigs/cloud-provider-azure/commit/cd295b03129579e5834a73cb41aa78a8d1aa1898

Reply:
Got it. I was not aware of the change.